Documentation/gettingstarted/cni-chaining-generic-veth.rst

```diff
@@ -0,0 +1,86 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    http://docs.cilium.io
+
+*********************
+Generic Veth Chaining
+*********************
+
+The generic veth chaining plugin enables CNI chaining on top of any CNI plugin
+that is using a veth device model which is the vast majority of plugins.
+
+Validate that the current CNI plugin is using veth
+==================================================
+
+1. Log into one of the worker nodes using SSH
+2. Run ``ip -d link`` to list all network devices on the node. You should be
+   able spot network devices representing the pods running on that node.
+3. A network device might look something like this:
+
+   .. code:: bash
+
+       103: lxcb3901b7f9c02@if102: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP mode DEFAULT group default qlen 1000
+           link/ether 3a:39:92:17:75:6f brd ff:ff:ff:ff:ff:ff link-netnsid 18 promiscuity 0
+           veth addrgenmode eui64 numtxqueues 1 numrxqueues 1 gso_max_size 65536 gso_max_segs 65535
+4. The ``veth`` keyword on line 3 indicates that the network device type is virtual ethernet.
+
+If the CNI plugin you are chaining with is currently not using veth then the
+``generic-veth`` plugin is not suitable. In that case, a full CNI chaining
+plugin is required which understands the device model of the underlying plugin.
+Writing such a plugin is trivial, contact us on Slack for more details.
```

Comment:
"...is trivial. Contact us on Slack for more details."

Perhaps add a link to the Slack as well.